mcp_servers/dropbox/README.md

```diff
@@ -0,0 +1,679 @@
+- **Remove File Member**: Remove collaborators from files
+- **Share Folder**: Share folders with team members
+- **List Folder Members**: View shared folder collaborators
+- **Add Folder Member**: Add members to shared folders
+
+### Account & Utilities
+- **Get Current Account**: Retrieve current user account information
+- **Get Space Usage**: Check storage space usage and allocation
+- **Get Temporary Link**: Generate temporary download links
+
+## Prerequisites
+
+### Dropbox App Permissions
+
+Before using this MCP server, you need to create a Dropbox app at [https://www.dropbox.com/developers/apps](https://www.dropbox.com/developers/apps) with the following permissions:
+
+#### Account Info
+- **account_info.read** - View basic information about your Dropbox account such as your username, email, and country
+
+#### Files and Folders
+- **files.metadata.write** - View and edit information about your Dropbox files and folders
+- **files.metadata.read** - View information about your Dropbox files and folders
+- **files.content.write** - Edit content of your Dropbox files and folders
+- **files.content.read** - View content of your Dropbox files and folders
+
+#### Collaboration
+- **sharing.write** - View and manage your Dropbox sharing settings and collaborators
+- **sharing.read** - View your Dropbox sharing settings and collaborators
+- **file_requests.write** - View and manage your Dropbox file requests
+- **file_requests.read** - View your Dropbox file requests
+- **contacts.write** - View and manage your manually added Dropbox contacts
+- **contacts.read** - View your manually added Dropbox contacts
+
+> **Note**: These permissions are required for the server to provide full functionality. Some features may not work if certain permissions are not granted.
+
+## Quick Start
+
+### 1. Setup Environment Variables
+
+Copy the example environment file and configure your Dropbox credentials:
+
+```bash
+cp .env.example .env
+```
+
+Edit `.env` with your Dropbox app credentials:
+
+```bash
+# Dropbox API Configuration
+DROPBOX_APP_KEY=your_app_key_here
+DROPBOX_APP_SECRET=your_app_secret_here
+DROPBOX_ACCESS_TOKEN=your_access_token_here
+```
+
+### 2. Install Dependencies
+
+```bash
+npm install
+```
+
+### 3. Start the Server
+
+```bash
+npm start
+```
+
+The server will validate your Dropbox connection on startup and display your account information.
+
+### 4. Use as HTTP Server
+
+Make requests with your access token (either from environment or header):
+
+```bash
+curl -X POST http://localhost:5000/mcp \
+  -H "Content-Type: application/json" \
+  -H "x-auth-token: YOUR_DROPBOX_ACCESS_TOKEN" \
+  -d '{"jsonrpc": "2.0", "id": 1, "method": "tools/list"}'
+```
+
+### 5. Use as MCP Server
+
+Add to your MCP client configuration:
+
+```json
+{
+  "servers": {
+    "dropbox": {
+      "command": "node",
+      "args": ["path/to/dropbox/mcp/server/dist/index.js"]
+    }
+  }
+}
+```
+
+## Available Tools
```

Comment:
I do not think you need to list all the tools here because it may be outdated if someone modifies the code. But your call.